src/ExtensionsSample/ErrorNotifier.cs

```diff
@@ -21,18 +18,9 @@ public class ErrorNotifier
         private const string NotificationUriSettingName = "AzureWebJobsErrorNotificationUri";
         private readonly string _webNotificationUri;
         private readonly HttpClient _httpClient = new HttpClient();
-        private readonly SendGridAPIClient _sendGrid;
```

Comment:
One thing I can do to group those changes is to remove each extension in distinct commits, allowing us to visualize the set of changes for each extension removed and either revert those specific commits for each extension as we start migration or at least use that for guidance.
I thought about commenting or adding conditionals to the code but that will litter the code base with those.